lib/Server/ResponseTypes/IdTokenResponse.php

```diff
@@ -97,7 +97,31 @@ protected function getExtraParams(AccessTokenEntityInterface $accessToken)
         $claims = $this->claimExtractor->extract($accessToken->getScopes(), $userEntity->getClaims());
 
         foreach ($claims as $claimName => $claimValue) {
-            $builder->withClaim($claimName, $claimValue);
+            switch ($claimName) {
+                case RegisteredClaims::AUDIENCE:
```

Comment:
`RegisteredClaim`  is missing an import/use  (.e.g `use Lcobucci\JWT\Token\RegisteredClaims;`)